Library/Formula/swig.rb

```diff
@@ -40,9 +40,14 @@ def install
       require "./test"
       puts Test.add(1, 1)
     EOS
+    headers = if ENV['HOMEBREW_RUBY'] == "1.8.7"
+      `ruby -rrbconfig -e 'print RbConfig::CONFIG["archdir"]'`
```

Comment:
Unfortunately not,

The ruby that is by default installed on OSX has its headers installed in `"/System/Library/Frameworks/Ruby.framework/Headers/"`
Ruby <= 1.8 headers are installed in `RbConfig::CONFIG["archdir"]`
Ruby >= 1.9 headers are installed in `RbConfig::CONFIG["rubyhdrdir"]`